### PR TITLE
Refactor Christian's codes (Parser, Redit, and Tagpull)

### LINKED ISSUE #316: Remove references to resuMeme
Should be ResuMe instead

### PR DESCRIPTION
**Pull request by chrisjwelly**
Mostly refactoring of long methods to modularise it into many smaller methods

Also fixes #316

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ResumeEditCommand.java
Comment (helloImHai) [inline src/main/java/seedu/address/logic/commands/ResumeEditCommand.java:104]: Nice separation

**Review by helloImHai — APPROVED** [review]
Looks good to me, but I think maybe hold this off until Hung's merges edit parser? Just incase there are any regressions.

File: src/main/java/seedu/address/logic/commands/ResumeEditCommand.java
Comment (chrisjwelly) [reply src/main/java/seedu/address/logic/commands/ResumeEditCommand.java:104]: Thanks m8

**Comment by chrisjwelly** [comment]
> 
> 
> Looks good to me, but I think maybe hold this off until Hung's merges edit parser? Just incase there are any regressions.

@helloImHai Agreed. Sure thing; this can wait.

File: src/main/java/seedu/address/logic/commands/edit/EditUserCommand.java
Comment (nhamhung) [inline src/main/java/seedu/address/logic/commands/edit/EditUserCommand.java:46]: Hehe Thanks

**Comment by nhamhung** [comment]
Yep it's LGTM!

**Pull request merged**
